scripts/regolith-sway-clamshell

```diff
@@ -40,42 +53,53 @@ get_action_cmd() {
   esac
 }
 
-INBUILT_DISPLAY=$( swaymsg -t get_outputs --raw | jq -r '[ .[] | select(.name | startswith("eDP")).name ] | .[0]' )
-
-# Early return if there is no inbuilt display
-if [[ -z $INBUILT_DISPLAY ]]; then
-  echo "No inbuilt display found. Exiting..."
-  exit 0
-fi
+handle_clamshell() {
+  if [[ "$ACTION_TYPE" == "LID_CLOSE" ]]; then
+    echo "clamshell mode: Turning off inbuilt display for lid close"
+    swaymsg output "$INBUILT_DISPLAY" disable
+  elif [[ "$ACTION_TYPE" == "LID_OPEN" ]]; then
+    echo "clamshell mode: Turning on inbuilt display for lid open"
+    swaymsg output "$INBUILT_DISPLAY" enable
+  fi
```

Comment:
suggestion: defensively, maybe guard that one of the two conditions of the compound predicate were executed with an else block that emits an error (or something).

```
else
  echo "Error: unhandled clamshell mode: $ACTION_TYPE"
fi
```